rqt_gui/package.xml

```diff
@@ -1,4 +1,4 @@
-<package>
+<package format="2">
   <name>rqt_gui</name>
   <version>0.5.0</version>
```

Comment:
Since the tags from different ROS distros (1 and 2) will share the same namespace we need room to grow the ROS 1 package versions in future releases. Therefore I would suggest using 1.0.0 for the ROS 2 branch.